datatypes/src/util/identifiers.rs

```diff
@@ -0,0 +1,77 @@
+pub trait Identifier: Sized {
+    /// Create a new (random) identifier
+    fn new() -> Self;
+}
+
+#[macro_export]
```

Comment:
>jdroenner commented on 20.01.2021
why do you need the macro? can't you do default implementations on the trait and/or implement it for T:xxx ?

because every identifier is a different type and they have nothing in common. I can't see a different way than using a macro

Reply:
if it's ok for you then I propose we leave it like it is for now

Reply:
ok leave it like this